### PR TITLE
update ACER hand-on doc

### PR DESCRIPTION
**Pull request by simonat2011**
add ACER hand-on doc

### CODE REVIEW

File: source/hands_on/acer.rst [hunk 1]
```diff
@@ -0,0 +1,288 @@
+ACER
+^^^^^^^
+
+Overview
+---------
+ACER, short for actor-critic with experience replay, is an off-policy actor-critic model with experience replay. It greatly increase
+the sample efficiency and decreasing the data correlation. ACER uses retrace Q-value estimation,  an efficient TRPO and truncates importance sample weights with
```
Comment (Will-Nie) [inline source/hands_on/acer.rst:7]: It greatly increases the sample efficiency and decreases the data correlation

File: source/hands_on/acer.rst [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    The first one is ``drop``, this means after splitting trajectory data into small pieces, we simply throw away those
+    with length smaller than ``unroll_len``. This method is kind of naive and usually is not a good choice. Since in
+    Reinforcement Learning, the last few data in an episode is usually very important, we can't just throw away them.
+
+    The second method is ``last``, which means if the total length trajectory is smaller than ``unrollen_len``,
+    we will use zero padding. Else, we will use data from previous piece to pad residual piece. This method is set as
+    default and recommended.
+
+    The last method ``null_padding`` is just zero padding, which is not vert efficient since many data will be ``null``.
+
```
Comment (Will-Nie) [inline source/hands_on/acer.rst:171]: 上边这一段关于如何处理采集的的训练数据写得非常全面,非常好. 但是这段其实每个算法都需要, 如果对于ACER, 我觉得可以适当精简一下, 不用占这么大的篇幅

File: source/hands_on/acer.rst [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+Now, we introduce the computation of vtrace-value.
+First, we use the following functions to compute retrace Q value.
+
```
Comment (Will-Nie) [inline source/hands_on/acer.rst:175]: 下面的方程要是像这样完全写出来(包括input, output的细节),我觉得可以简单放一下所在目录, 因为认真读的人一般都会想去看一下每部分怎么连接的. 另外一种方式是直接放与上面图片相对应的核心代码, 忽略input/output等

File: source/hands_on/acer.rst [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+        actions=actions.unsqueeze(-1)
+        with torch.no_grad():
+            advantage_retraces = q_retraces-v_pred
+            advantage_native = q_values-v_pred
+        actor_loss = ratio.gather(-1,actions).clamp(max=c_clip_ratio)*advantage_retraces*(target_pi.gather(-1,actions)+EPS).log()
+        bc_loss = (1.0-c_clip_ratio/(ratio+EPS)).clamp(min=0.0)*target_pi.detach()*advantage_native*(target_pi+EPS).log()
+        bc_loss=bc_loss.sum(-1).unsqueeze(-1)
+        return actor_loss,bc_loss
+
+Then, we execute backward operation until target_pi. The we need to calculate the correction gradient in the trust region:
```
Comment (Will-Nie) [inline source/hands_on/acer.rst:237]: Moreover, we need to calculate the correction gradient in the trust region:

Comment (Will-Nie) [inline source/hands_on/acer.rst:237]: towards target_pi?
